### DISCUSSION TITLE
Need to suggestions in both find option Editor finding and Global Finding

### DISCUSSION THREAD

**Discussion by indrajeet4192** [microsoft/vscode-discussions#623] [Extension Development QnA] [answered]
Whenever we need to find something in VS code, we have either copy text or type manually, but if we have suggestion for recent searches and suggestions for find which come in ctrl + f and in ctrl+shift+f it will consume less time and developer can work faster.

I hope you got this.

**Comment by alefragnani** [comment, marked as answer]
Hi @indrajeet4192 ,

It already does that. 
* To use previous searches, both in Find / Find in Files, use the <kbd>Up Arrow</kbd> key. 
* In Find in Files, the _currently selected text in the editor_ will be used in new searches
* In Find, you can customize to use selected text, word at cursor or nothing at all, using `editor.find.seedSearchStringFromSelection` setting

By the way, for future feature requests about VS Code usage, please use the VS Code repository instead (https://github.com/microsoft/vscode/issues/new/choose). This Discussions repo is focused for extension development, for now..

Hope this helps
